icanbwell/fhir-server-ui
# EFS-698 Moved authentication from express to react

- Moved authentication from express to react, also removed all auth URLs from express server
- Fixed pagination reload bug .i.e broken pagination if we reload the page with URL containing _getpagesoffset param

Steps to test the authentication flow:
- Clone the repo
- Run `make up`
- Visit `http://localhost:5051` this link should not require authentication
- Visit `http://localhost:5051/4_0_0/Person` this link should redirect to cognito auth page
- Enter username and password
- You should be redirected to `http://localhost:5051/4_0_0/Person` and Person resources should load
- Now, click the logout button on top right, this should redirect you to `http://localhost:5051` and you will have to login again
